source/platforms/_tests/fixtures/github_pr.json

```diff
@@ -127,21 +127,21 @@
         "site_admin": false
       },
       "private": false,
-      "html_url": "https://github.com/artsy/emission",
+      "html_url": "https://github.com/orta",
       "description": "React Native Components",
       "fork": false,
-      "url": "https://api.github.com/repos/artsy/emission",
-      "forks_url": "https://api.github.com/repos/artsy/emission/forks",
-      "keys_url": "https://api.github.com/repos/artsy/emission/keys{/key_id}",
-      "collaborators_url": "https://api.github.com/repos/artsy/emission/collaborators{/collaborator}",
-      "teams_url": "https://api.github.com/repos/artsy/emission/teams",
-      "hooks_url": "https://api.github.com/repos/artsy/emission/hooks",
-      "issue_events_url": "https://api.github.com/repos/artsy/emission/issues/events{/number}",
-      "events_url": "https://api.github.com/repos/artsy/emission/events",
-      "assignees_url": "https://api.github.com/repos/artsy/emission/assignees{/user}",
-      "branches_url": "https://api.github.com/repos/artsy/emission/branches{/branch}",
-      "tags_url": "https://api.github.com/repos/artsy/emission/tags",
-      "blobs_url": "https://api.github.com/repos/artsy/emission/git/blobs{/sha}",
+      "url": "https://api.github.com/repos/orta",
+      "forks_url": "https://api.github.com/repos/orta/emission/forks",
+      "keys_url": "https://api.github.com/repos/orta/emission/keys{/key_id}",
+      "collaborators_url": "https://api.github.com/repos/orta/emission/collaborators{/collaborator}",
+      "teams_url": "https://api.github.com/repos/orta/emission/teams",
+      "hooks_url": "https://api.github.com/repos/orta/emission/hooks",
+      "issue_events_url": "https://api.github.com/repos/orta/emission/issues/events{/number}",
+      "events_url": "https://api.github.com/repos/orta/emission/events",
+      "assignees_url": "https://api.github.com/repos/orta/emission/assignees{/user}",
+      "branches_url": "https://api.github.com/repos/orta/emission/branches{/branch}",
+      "tags_url": "https://api.github.com/repos/orta/emission/tags",
+      "blobs_url": "https://api.github.com/repos/orta/emission/git/blobs{/sha}",
       "git_tags_url": "https://api.github.com/repos/artsy/emission/git/tags{/sha}",
       "git_refs_url": "https://api.github.com/repos/artsy/emission/git/refs{/sha}",
       "trees_url": "https://api.github.com/repos/artsy/emission/git/trees{/sha}",
```

Comment:
Just arbitrary changed the head branch to be different from the base branch so that tests can provide which one they use, TBH - should have called it 'head' - but I need to get all these balls in the air - so no time this second.